projects/ad6676-ebz/src/ad6676_ebz.c

```diff
@@ -429,8 +429,6 @@ int main(void)
 	axi_dmac_transfer(ad6676_dmac, ADC_DDR_BASEADDR, 16384 * 2);
 
 #ifdef IIO_SUPPORT
-	printf("The board accepts libiio clients connections through the serial backend.\n");
```

Comment:
Some of this projects are using a "local" (inside src) version of iio_app. Some haven't been updated to use iio/iio_app
Maybe is not yet the moment to remove them

Reply:
yeah i saw that
the reasoning was that currently that message is incorrectly displayed anyway in projects that change the sampling rate, user now sees something like "The board [junk characters]"

But I am ok with dropping the 2nd commit and have each project adopt iio_app gradually